site/content/docs/Localisation/_index.md

```diff
@@ -82,10 +83,46 @@ TBD
 }
 ```
 
-### Tests
-```kopiaUI``` features different tests to prevent situations where translations become obsolet and or unused. The tests check for keys with empty or null values and that keys exist in all provided translation files. The tests will fail if translation files contain empty keys or if keys are not declared in all files. 
+### Naming Conventions
+We defined a strict naming convention to facilitate a meaningful and extendable organization of translations across ```kopiaUI```and ```kopia``` in the future. A proper naming convention is needed to provide context and hints to translators - lowering the chances of misunderstandings and poor translations. The naming convention uses the following schema: ```[type].[category].[sub-category].[id]```
+
+There are 
+
+| Type  | Meaning  |
+|---|---|
+|```feedback``` | The type ```feedback``` shoule be used when informing or displaying information to the user. For example, these can be hints, placeholders or labels |
+|```value```    | The type ```value``` is used to |
+|```event```    |   |
+|```common```   |   |
+
+
+| Category  | Meaning  |
+|---|---|
+|```policy```    | The type ```feedback``` shoule be used when informing or displaying information to the user. For example, these can be hints, placeholders or labels |
+|```repository```       | The type ```repository``` is used to    |
+|```task```      |   |
+|```log```     |   |
+|```pin```     |   |
+|```validation```     |   |
+|```button```     |   |
+|```label```     |   |
+|```provider```     |   |
+|```algorithm```     |   |
+|```ui```     |   |
+|```ui```     |   |
 
-Additionally, the test extracts the currently used translation keys within the code to check if all keys are used at least once. 
+
+| Sub-Category  | Meaning  |
+|---|---|
+|```policy```    | The type ```feedback``` shoule be used when informing or displaying information to the user. For example, these can be hints, 
+
+
+
+### Namespaces
+TBD
+
+### Tests
+```kopiaUI``` features different tests to prevent situations where translations become obsolet and or unused. The tests check for keys with empty or null values and that keys exist in all provided translation files. The tests will fail if translation files contain empty keys or if keys are not declared in all files. Additionally, the test extracts the currently used translation keys within the code to check if all keys are used at least once. 
```

Comment:
```suggestion
```kopiaUI``` features different tests to prevent situations where translations become obsolete and or unused. The tests check for keys with empty or null values and that keys exist in all provided translation files. The tests will fail if translation files contain empty keys or if keys are not declared in all files. Additionally, the test extracts the currently used translation keys within the code to check if all keys are used at least once. 
```